### PR TITLE
feat: add real location for planter pages

### LINKED ISSUE #1060: Add real location for planters page
---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by khalatevarun**
# Description

[comment]: # 'Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.'

Fixes #1060 

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

|       Before        |       After        |
| :-----------------: | :----------------: |
|SCR-20221005-6ns|SCR-20221005-6pu|

# How Has This Been Tested?

- [ ] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/pages/planters/[planterid].js [hunk 1]
```diff
@@ -242,7 +242,10 @@ export default function Planter(props) {
                 />
               </Box>
               <Box sx={{ mt: 2 }}>
-                <Info iconURI={LocationIcon} info="Shirimatunda, Tanzania" />
+                <Info
+                  iconURI={LocationIcon}
+                  info={`${planter.country_name}, ${planter.continent_name}`}
```
Comment (RubenSmn) [inline src/pages/planters/[planterid].js:247]: @dadiorchen can we assume the planter always has these fields or do we need to add some checks?

Comment (khalatevarun) [reply src/pages/planters/[planterid].js:247]: what is the fallback value that can be added?

Comment (RubenSmn) [reply src/pages/planters/[planterid].js:247]: The fallback should probably be "Unknown"

Comment (khalatevarun) [reply src/pages/planters/[planterid].js:247]: should i add fallback for both separately?

> info={`${planter.country_name ?? 'Unknow'}, ${planter.continent_name ?? 'Unknown'}`}

**Comment by khalatevarun** [comment]
@RubenSmn you can check now, i have added fallback

**Review by RubenSmn — CHANGES_REQUESTED** [review]
Same as with the #1059 I think we need a more general approach here;
- if both present: `Country, Continent`
- if only one present: `Country` or `Continent`
- if none present: `Unknown`

Maybe we could create a function in `utils` to take care of this logic

**Review by RubenSmn — APPROVED** [review]

**Pull request merged**